tests/Rules/PHPUnit/ClassMethodCoversExistsRuleTest.php

```diff
@@ -26,15 +26,15 @@ public function testRule(): void
 	{
 		$this->analyse([__DIR__ . '/data/method-coverage.php'], [
 			[
-				'@covers value ::ignoreThis references an invalid class.',
+				'@covers value ::ignoreThis references an invalid method.',
```

Comment:
these cases changed because the messaging was refined to indicate it's a method if `::` is found in the `@covers` annotation,

previously it would say `class` in all cases which was a little misleading

Reply:
What would be great is to add the nonexistent names to all these messages. Be as explicit as possible 😊

Reply:
as in the fully qualified name? i did consider adding that but didn't want to change too much at once!

Reply:
You can change it, it's in bleedingEdge 😊